src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -8,16 +8,16 @@ namespace Microsoft.Framework.FileSystemGlobbing.Abstractions
 {
     public class DirectoryInfoWrapper : DirectoryInfoBase
     {
-        private DirectoryInfo DirectoryInfo;
+        private readonly DirectoryInfo _directoryInfo;
 
         public DirectoryInfoWrapper(DirectoryInfo directoryInfo)
         {
-            DirectoryInfo = directoryInfo;
+            _directoryInfo = directoryInfo;
         }
 
         public override IEnumerable<FileSystemInfoBase> EnumerateFileSystemInfos(string searchPattern, SearchOption searchOption)
         {
-            foreach (var fileSystemInfo in DirectoryInfo.EnumerateFileSystemInfos(searchPattern, searchOption))
+            foreach (var fileSystemInfo in _directoryInfo.EnumerateFileSystemInfos(searchPattern, searchOption))
```

Comment:
How do we treat files starting with `.` or `hidden` or `system`. This originally came up in security discussion https://github.com/aspnet/FileSystem/issues/51 @GrabYourPitchforks @glennc